src/nvim/syntax.c

```diff
@@ -5912,6 +5912,7 @@ static char *highlight_init_both[] =
   "default link EndOfBuffer NonText",
   "default link QuickFixLine Search",
   "default link Substitute Search",
+  "default link WhiteSpace NonText",
```

Comment:
I agree with @zhou13 even if someone complains.  I never will think "I want to see invisible characters, but I want them to be highlighted to look like editable, opaque text because I want to confuse myself and I also want `:map` to be difficult to read."